crates/ruff/src/rules/flake8_pyi/rules/bad_version_info_comparison.rs

```diff
@@ -0,0 +1,83 @@
+use rustpython_parser::ast::{Cmpop, Expr};
+
+use ruff_macros::{define_violation, derive_message_formats};
+
+use crate::checkers::ast::Checker;
+use crate::registry::Diagnostic;
+use crate::violation::Violation;
+use crate::Range;
+
+define_violation!(
+    /// ## What it does
+    /// Checks for usages of comparators other than `<` and `>=` for
```

Comment:
I just removed the empty lines here for consistency with the other docs.

Reply:
ok i'll switch to that style